vllm/engine/arg_utils.py

```diff
@@ -923,7 +923,13 @@ def add_cli_args(parser: FlexibleArgumentParser,
         parser.add_argument('--engine-use-ray',
                             action='store_true',
                             help='Use Ray to start the LLM engine in a '
-                            'separate process as the server process.')
+                            'separate process as the server process.'
+                            '(DEPRECATED. This argument is deprecated '
+                            'and may be removed in a future update. '
```

Comment:
suggest more decisive wording :)
```suggestion
                            'and will be removed in a future update. '
```